### ISSUE TITLE
Add DPIA/CAG info to documentation

### ISSUE THREAD

**Issue by nikyraja** [rcpch/rcpch-audit-engine#625]
Hello, 

Please could you add the DPIA to the documentation? 

@Marcus, could you first please double check the information on the risk tab on page 26 - 27 and let me know if you have any edits (or to anything else!)? If we could then upload as a pdf please.
[PIA 2018-02 Epilepsy12 PIA v2.1_091123.docx](https://github.com/rcpch/rcpch-audit-engine/files/13311103/PIA.2018-02.Epilepsy12.PIA.v2.1_091123.docx)

We can also add the CAG information if not already done: 
1. Section 251 annual review
[17CAG0184 Change in data processor - Annual review outcome letter.pdf](https://github.com/rcpch/rcpch-audit-engine/files/13311110/17CAG0184.Change.in.data.processor.-.Annual.review.outcome.letter.pdf)

2. NDO exemption
[17CAG0184 NDO deferral request conditionally supported outcome.pdf](https://github.com/rcpch/rcpch-audit-engine/files/13311112/17CAG0184.NDO.deferral.request.conditionally.supported.outcome.pdf)

3. Amendment for change in data platform
[17CAG0184 Fully Supported Amendment Outcome (1).pdf](https://github.com/rcpch/rcpch-audit-engine/files/13311113/17CAG0184.Fully.Supported.Amendment.Outcome.1.pdf)

**Comment by pacharanero** [comment]
Hi @nikyraja I have made a few amendments (mostly small typos) to the DPIA and added some comments.
[pia-2018-02-epilepsy12-pia-v2-1_091123.docx](https://github.com/rcpch/rcpch-audit-engine/files/13500520/pia-2018-02-epilepsy12-pia-v2-1_091123.docx)

Please could you review and resolve comments, creating Issues as necessary for any outstanding development backlog.
Once done if we convert the .docx into a PDF for publication this is probably better.
I can add the other documents as well at that stage

**Comment by nikyraja** [comment]
Thanks Marcus, I've responded to the queries on this doc: 
[pia-2018-02-epilepsy12-pia-v2-1_091123_NR (1).docx](https://github.com/rcpch/rcpch-audit-engine/files/13502927/pia-2018-02-epilepsy12-pia-v2-1_091123_NR.1.docx)
- The 1st was an issue where tracked changes hadn't recorded correctly. Have resolved this, thanks for flagging. 
- Have responded on the hospital identifier as well.
- I've also updated the doc to reflect that we are still working on database encryption.

Here is the final version as a pdf
[PIA-2018-02-epilepsy12-pia-v2-1_Final.pdf](https://github.com/rcpch/rcpch-audit-engine/files/13502552/PIA-2018-02-epilepsy12-pia-v2-1_Final.pdf)

Happy for this to go on the doc site.
